JS-Algorithms/challenges.ts

```diff
@@ -12,8 +12,31 @@ Example:
 Invoking "readableTime(3690)" should return "01:01:30" (HH:MM:SS)
 ***** */
 
-const readableTime = (seconds) => {
-  // YOUR CODE HERE...
+const readableTime = (seconds: number): string => {
+  let hours: number = Math.floor(seconds / 3600),
+    minutes: number = Math.floor((seconds - hours * 3600) / 60),
+    secs: number = seconds - hours * 3600 - minutes * 60;
+
+  let strHours: string = "",
```

Comment:
Could you assign these variables when they are declared vs assigning them later? 

You could use a ternary to cut this amount of code down

`const strHours =  hours < 10 ? "0" + hours : ""`